install.sh

```diff
@@ -302,27 +303,34 @@ yes | crew install core
 echo_info "\nRunning Bootstrap package postinstall scripts...\n"
 crew postinstall $BOOTSTRAP_PACKAGES
 
-echo_info "Synchronizng local package repo..."
-# First clear out temporary package repo so we can replace it with the 
-# repo downloaded via git.
-find "${CREW_LIB_PATH}" -mindepth 1 -delete
-
-# Do a minimal clone, which also sets origin to the master/main branch
-# by default. For more on why this setup might be useful see:
-# https://github.blog/2020-01-17-bring-your-monorepo-down-to-size-with-sparse-checkout/
-# If using alternate branch don't use depth=1 .
-[[ "$BRANCH" == "master" ]] && GIT_DEPTH="--depth=1" || GIT_DEPTH=
-git clone $GIT_DEPTH --filter=blob:none --no-checkout "https://github.com/${OWNER}/${REPO}.git" "${CREW_LIB_PATH}"
-
-cd "${CREW_LIB_PATH}"
-
-# Checkout, overwriting local files.
-[[ "$BRANCH" != "master" ]] && git fetch --all
-git checkout "${BRANCH}"
-
-# Set sparse-checkout folders.
-git sparse-checkout set packages lib bin crew tools
-git reset --hard origin/"${BRANCH}"
+if ! "${CREW_PREFIX}"/bin/git version &> /dev/null; then
+  echo_other "\nGit is broken on your system, and crew update will not work properly."
+  echo_other "Please report this here:"
+  echo_other "https://github.com/chromebrew/chromebrew/issues\n"
```

Comment:
It's great we're adding more color options but shouldn't this be `echo_error`?

Reply:
Yeah, it should.